operators/churner/metrics.go

```diff
@@ -76,6 +89,15 @@ func (g *Metrics) ObserveLatency(method string, latencyMs float64) {
 	g.Latency.WithLabelValues(method).Observe(latencyMs)
 }
 
+// IncrementRequestNum increments the number of successful requests
+func (g *Metrics) IncrementRequestNum(method string) {
+	g.NumRequests.With(prometheus.Labels{
+		"status": "total",
```

Comment:
Same comment here, feel like we shouldn't track total because we can compute it from failed + success.